extra_data/reader.py

```diff
@@ -327,6 +327,10 @@ def trains(self, devices=None, train_range=None, *, require_all=False,
             iteration indexed by source and then key. True returns a
             flat dictionary indexed by (source, key) tuples.
 
+        keep_dims: bool
+            False (default) based on count index or slice the data.
+            True returns a sliced data to preserve the dimensions.
```

Comment:
```suggestion
            False (default) drops the first dimension when there is
            a single entry. True preserves this dimension.
```

Reply:
Thanks this is indeed more understandable.